packages/analytics-js/src/components/configManager/util/commonUtil.ts

```diff
@@ -0,0 +1,34 @@
+import { Nullable } from '@rudderstack/analytics-js/types';
+
+/**
+ * Removes trailing slash from url
+ * @param url
+ * @returns url
+ */
+const removeTrailingSlashes = (url: string | null): Nullable<string> =>
```

Comment:
please move this one to components utilities string file

Reply:
Done